javascriptv3/example_code/dynamodb/README.md

```diff
@@ -1,21 +1,28 @@
-# Amazon DynamoDB JavaScript SDK v3 code examples
+# Amazon DynamoDB code examples for AWS SDK for JavaScript (v3)
```

Comment:
DynamoDB code examples for the SDK for JavaScript (v3)

Reply:
done